Fix a jsoncpp submodule typo

peternewman opened the pull request.

kripton reviewed (approved):
Thanks for finding, this is indeed a funny one. I'm on mobile right now but on line 19 of the .gitmodules (submodule name), the typo is still there. Can you rename that as well and check if anything breaks? Thanks! Apart from that, good to merge

peternewman commented:
> Thanks for finding, this is indeed a funny one. I'm on mobile right now but on line 19 of the .gitmodules (submodule name), the typo is still there. Can you rename that as well and check if anything breaks? Thanks! Apart from that, good to merge

Dammit, I fixed that originally in https://github.com/OpenLightingProject/rp2040-dmxsun/pull/66/commits/11916fbbcf848c8db09a5c3be8436c18340efb12 but it failed to update the folder names themselves and failed, so I then did it properly on the CLI and it seems it's missed that now!

Now fixed anyway...

kripton commented:
Indeed, changing stuff around submodules can be tedious and confusing. Nevertheless, nice catch, thanks for the PR!
